keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -51,16 +48,18 @@ Consistent reads may be served from cache so long as:
 - The data in the watch cache no older than the latest "revision" just from etcd
 
 etcd watches support "progress events", which provide an updated revision and a
-guarantee that all future watch events will be newer than the that revision.  If
-an etcd watch is configured with `WithProgressNotify` enabled, etcd
-automatically sends progress events at a regular interval. The "progress events"
-allow a etcd watcher to know how up-to-date the watch stream is relative a
-particular revision.
+guarantee that all future watch events will be newer than the that revision. 
+Etcd client can request a progress notification from server. The progress 
+notification allow the etcd watcher to know how up-to-date the watch stream 
+is. This is thanks to [bookmarkable] property of etcd watch that guarantees that
+all events with revision below progress notification have been delivered.
 
 This KEP summarizes how we can take advantage of progress events efficiently
 determine how up-to-date kubernetes watch caches are then serve reads from the
 watch cache when they are sufficiently up-to-date.
 
+[bookmarkable]: https://etcd.io/docs/v3.6/learning/api_guarantees/#watch-apis
```

Comment:
In which versions of etcd https://github.com/etcd-io/etcd/issues/15220 was fixed?

Reply:
v3.5.8 and v3.4.25 that are 2 months old releases.